packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -51,7 +51,9 @@ function ChoiceButton({
 
   return (
     <Button
+      a11yTitle={choiceLabel}
       ref={choiceButton}
+      as='li'
```

Comment:
List items aren’t focusable or clickable. The list of buttons should be marked up as a list, but the buttons should be inside the list items.